content_schemas/formats/shared/default_properties/publisher.jsonnet

```diff
@@ -23,6 +23,10 @@
     format: "date-time",
     description: "Last time when the content received a major or minor update.",
   },
+  editor_id: {
+    type: "string",
+    description: "The UUID of the editor who updated the content.",
```

Comment:
**question:** intended use

The commit message now says "This takes the UUID of the user who _created_ the edition." while the description says "The UUID of the editor who _updated_ the content." Is it meant to be the creator or the last editor? I wonder if the field name could be more explicit, like before, e.g. `last_edited_by_id` or `last_edited_by_uuid`?

Reply:
I've gone with `last_edited_by_editor_id`, which also works well with the `last_edited_at` attribute